packages/react-native/src/surveys/components/ConfirmationMessage.tsx

```diff
@@ -24,11 +24,17 @@ export function ConfirmationMessage({
 }): JSX.Element {
   const textColor = getContrastingTextColor(appearance.backgroundColor)
 
+  const processedDescription = description
+    ? contentType === SurveyQuestionDescriptionContentType.Html
+      ? description.replace(/<[^>]*>/g, '') // Strip HTML tags for React Native
+      : description
+    : null
+
```

Comment:
```
const processDescription = (description: string | null, contentType: SurveyQuestionDescriptionContentType) => {
  if (!description) return null;
  
  return contentType === SurveyQuestionDescriptionContentType.Html
    ? description.replace(/<[^>]*>/g, '')
    : description;
};

const processedDescription = processDescription(description, contentType);
```

minor suggestion to define a shared function since it's used in the QuestionHeader too. this function would live in a utils file somewhere